licenses/models.py

```diff
@@ -342,4 +342,30 @@ class TranslationBranch(models.Model):
     last_transifex_update = models.DateTimeField(
         "Time when last updated on Transifex.", null=True, blank=True,
     )
-    complete = models.BooleanField("Only one incomplete per branch", default=False,)
+    complete = models.BooleanField(default=False)
+
+    class Meta:
+        verbose_name_plural = "translation branches"
+
+    def __str__(self):
+        return f"Translation branch {self.branch_name}. {'Complete' if self.complete else 'In progress'}."
+
+    @property
+    def stats(self):
+        untranslated_messages = 0
+        translated_messages = 0
+        for code in self.legalcodes.all():
+            pofile = code.get_pofile()
+            untranslated_messages += len(pofile.untranslated_entries())
+            translated_messages += len(pofile.translated_entries())
+        total_messages = untranslated_messages + translated_messages
+        if total_messages:
+            percent = int(translated_messages * 100 / float(total_messages))
+        else:
+            percent = 100
```

Comment:
if `total_messages` is `0` we will hit this `else block`. Shouldn't `percent` be `0` or `None`.

Reply:
My first thought was to use 0 too, but I feel like either 0 or 100 could be valid.  (There are no untranslated messages, so we must be 100% translated.)

I went with 100 to be consistent with what ```polib.POFile``` returns as ```percent_translated``` when there are no messages. That's what we display on the branch status page.

On reflection, though, maybe we should just display "-" in the boxes when there are no messages, rather than trying to come up with a number.